README.md

```diff
@@ -10,4 +10,39 @@ cd my-app
 npm init solid@next
 npm install
 npm run dev
-```
\ No newline at end of file
+```
```

Comment:
https://github.com/solidjs/solid-start/pull/5#issuecomment-840304696
> To be honest I sort of intend to change how to set the entry. Relying on /node_modules is super hacky. I was doing it for now but I expect this to change before even we reach beta. I can add this for now. But more than likely I will remove it in the next couple weeks.

I wish there was a "git blame" for the future, i.e. what changed this code next?

For now we'll check manually:
1. Search docs ("README.md") for "monorepo": https://github.com/solidjs/solid-start/blame/main/README.md

Conclusion: Monorepo docs are here: https://github.com/solidjs/solid-start#monorepo--projectjson-workspace-support :)